MHCollapsibleViewManager.h

```diff
@@ -15,9 +15,31 @@
 #import <UIKit/UIKit.h>
 #import "MHFilterLabel.h"
 #import "MHTableViewCell.h"
+#import "MHCollapsibleSection.h"
 
+@protocol MHCollapibleViewManagerDelegate <NSObject>
+
+@required
+
+- (void) createModalWithType:(CRUCellViewInteractionType)cellType section:(MHCollapsibleSection*)section row:(NSUInteger)row;
+
+@end
+
+@interface MHCollapsibleViewManagerDeletegate : NSObject{
+    
+    id <MHCollapibleViewManagerDelegate> _delegate;
```

Comment:
You can remove this. Your property statement can take care of defining your delegate ie
```
@interface MHCollapsibleViewManagerDeletegate : NSObject
@property (nonatomic, weak) id<MHCollapibleViewManagerDelegate> delegate;
...
@end
```
Note that a delegate needs to have a retain type of weak so that you don't get into a loop where the collapsible view manager is being retained by the object that created it but the collapsible view manager also retains the object that created it (because in most cases the object that created it is also the delegate). If they both retain one another the objects will be done with, the garbage collector will come along to try and remove them from memory but even though that no part of the app is still using either they are retaining each other. The garbage collector will leave both in memory for the life time of the app.